fossilize.cpp

```diff
@@ -6927,6 +6993,35 @@ static bool json_value(const VkRenderPassInputAttachmentAspectCreateInfo &create
 	return true;
 }
 
+template <typename Allocator>
+static bool json_value(const VkPipelineDiscardRectangleStateCreateInfoEXT &create_info, Allocator &alloc, Value *out_value)
+{
+	Value value(kObjectType);
+	value.AddMember("sType", create_info.sType, alloc);
+	value.AddMember("flags", create_info.flags, alloc);
+	value.AddMember("discardRectangleMode", create_info.discardRectangleMode, alloc);
+	value.AddMember("discardRectangleCount", create_info.discardRectangleCount, alloc);
```

Comment:
Or hm ... Perhaps we need separate count variable if discard rectangle is considered dynamic?

Reply:
Yes, we store viewportCount/scissorCount for this reason, so ...